docs/start.rst

```diff
@@ -67,4 +73,38 @@ A good place to get started with with your own programs might be to copy `exampl
 
 .. todo::
 
-    discuss the generated output files
\ No newline at end of file
+    discuss the generated output files
+
+
+Notebooks
+---------
+
+Following Jupyter notebooks are available in ``<OpenQL Root Dir>/examples/notebooks`` directory:
```

Comment:
notebooks are available in *the* ``<OpenQL Root Dir>/examples/notebooks`` directory

Add "the"  in multiple places

Reply:
fixed